docs/csharp/language-reference/compiler-messages/cs2032.md

```diff
@@ -47,10 +47,10 @@ Character 'character' is not allowed on the command-line or in response files
   
 2.  In the **My Documents** folder, create a text file that is named cs2032.cs and that contains whatever you want.  
   
-3.  Open the **Start** menu. Locate and select the [Developer Command Prompt for Visual Studio](https://msdn.microsoft.com/library/ms229859(v=vs.110).aspx).
+3.  Open the **Start** menu. Locate and select the [Developer Command Prompt for Visual Studio](https://docs.microsoft.com/dotnet/framework/tools/developer-command-prompt-for-vs).
```

Comment:
I think we should do this on another branch/PR ... I'm about to blow up GH here. :boom: ... 🚒 🚑 🚓